### ISSUE TITLE
Unable to download Godot 1.0

### ISSUE THREAD

**Issue by BNTFryingPan** [godotengine/godot-website#733]
I am currently making a tool that downloads Godot versions, and decided to try downloading v1.0 to test it. It failed so I went to see if it was actually on the archive page, and while it is listed, none of the downloads work as the actual GitHub release for 1.0 doesn't have any binaries attached to it on either the actual engine repo, or the godot-builds repo. They also don't exist on tuxfamily either for some reason.

I don't think Godot 1.0 should be listed just like all the other versions with download buttons if the downloads do not work, so either binaries should be added to the above places, or the website should have some indication that you need to compile it yourself or at least that downloads are not available for whatever reason.

**Comment by YuriSizov** [comment]
Yes, the official Godot 1.0 binaries are lost to time and are not available for download. It would be nice to reproduce them at some point, but that's a bit of a tall order.

Removing links specifically for 1.0 would require some hacking around the page generator which is not worth it for something that currently only has historical interest, but no practical application. So it will remain as is until we can build it.

**Comment by Calinou** [comment]
You can find unofficial 1.0 builds here: https://github.com/Calinou/godot-vintage-builds

**Comment by coppolaemilio** [comment]
@Calinou could we use your unofficial builds for the website? It would be nice to provide at least yours

**Comment by Calinou** [comment]
> @Calinou could we use your unofficial builds for the website? It would be nice to provide at least yours

Sure 🙂 

I could open a PR for it, although note that I don't have macOS builds and building even for x86_64 macOS will prove challenging.

My builds are also not codesigned, but that's probably not critical given the niche use case.

**Comment by coppolaemilio** [comment]
You can download them now. The only one missing is mac, but there should be another issue for it. Closing this one! https://godotengine.org/download/archive/1.0-stable/
